modules/standard/Spawn.chpl

```diff
@@ -253,6 +253,9 @@ module Spawn {
       }
     }
 
+    /*
+      :throws SystemError: If the subprocess does not launch
+    */
 
     pragma "no doc"
```

Comment:
Note that this function is marked with `pragma "no doc"`.  That means that it will not turn up in the documentation normally, so doesn't need to have a documentation comment with `:throws:` in it.

Reply:
I hadn't had time on the issue to go through an eliminate false positives.  Hopefully this helps avoid tripping over them in the future!

Reply:
I was having this confusion while going through the file and was about to ask you about it.
I'll remove this part.